web/client/codechecker_client/cmd_line_client.py

```diff
@@ -419,7 +420,14 @@ def handle_list_results(args):
             if status is not None:
                 dt_status = ttypes.DetectionStatus._VALUES_TO_NAMES[status]
 
-            rows.append((checked_file, res.checkerId, sev, res.checkerMsg,
+            # Remove new line characters from the checker message.
+            msg = ' '.join(res.checkerMsg.split())
```

Comment:
Wouldn't be better using `re.sub()` ?

Reply:
Okay, now I am using this function and I am also removing all the whitespace characters (`\s+`) from the checker message.